src/CSET/recipes/generic_plevel_spatial_plot_sequence.yaml

```diff
@@ -0,0 +1,25 @@
+category: Quick Look
+title: $VARNAME $PLEVEL Level Spatial Plot
```

Comment:
We could probably include the recipe title in the plot title, which would ensure we had useful information in there.

Reply:
See #457 for this.